src/test/java/com/sap/sgs/phosphor/fosstars/data/StandardValueCacheTest.java

```diff
@@ -36,9 +43,63 @@ public void store() throws IOException {
       assertTrue(content.length > 0);
 
       StandardValueCache anotherCache = StandardValueCache.load(filename);
-      assertEquals(cache, anotherCache);
+      assertEquals(cache, cache);
```

Comment:
assertEquals(cache, anotherCache);

Reply:
This line checks that `cache` equals to itself. The next line checks if `cache` equals to `anotherCache`.